index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="stylesheet" href="style.css">
+  <title>Personal Portfolio</title>
+</head>
+<body>
+  <!--Hero Section-->
+  <section id="hero">
+
+    <header>
+      <nav id="navbar">
+        <div class="hamburger">
+          <a href="#"><i class="fa fa-bars"></i></a>
+        </div>
+      </nav>
+    </header>
+
+    <div class="banner">
+
+      <div class="introduction">
+
+        <div class="title">
+          <div class="self">Hey there.</div>
+          <div class="self">I'm Alex.</div>
+          <div class="profession">I’m a software developer</div>
+        </div>
+        <div class="info">
+          <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hesitate to contact me.
+          </p>
+        </div>
+        <div class="social">
+          <img src="./image/twitter.png" alt="twitter icon" class="twitter">
+          <img src="./image/Linkedin icon.png" alt="Linkedin icon" class="linkedin">
+          <img src="./image/medium.png" alt="Medium icon" class="medium">
+          <img src="./image/github.png" alt="Github icon" class="github">
```

Comment:
Well done here as well your icons are correctly aligned 👌 I suggest just that you wrap each img in an anchor a tag as it is meant to be a link that redirects the user to your social media accounts also it's a good practice to use ul element for list of items for semantic reasons